Add page for HTMLDataListElement.options (#35662)
mdn/content: files/en-us/web/api/htmldatalistelement/options/index.md

---
title: "HTMLDataListElement: options property"
short-title: options
slug: Web/API/HTMLDataListElement/options
page-type: web-api-instance-property
browser-compat: api.HTMLDataListElement.options
---

{{APIRef("HTML DOM")}}

> The **`options`** read-only property of the {{domxref("HTMLDataListElement")}} interface returns an {{domxref("HTMLCollection")}} of {{domxref("HTMLOptionElement")}} elements contained in this datalist. These descendant options are used to provide predined options for the input control associated with the datalist.

Reviewer: ```suggestion
The **`options`** read-only property of the {{domxref("HTMLDataListElement")}} interface returns an {{domxref("HTMLCollection")}} of {{domxref("HTMLOptionElement")}} elements contained in a {{htmlelement("datalist")}}. The descendant {{htmlelement("option")}} elements provide predefined options for the {{htmlelement("input")}} control associated with the `<datalist>`.
```

## Value

An {{domxref("HTMLCollection")}} of {{domxref("HTMLOptionElement")}} elements.

## Specifications

{{Specifications}}

## Browser compatibility

{{Compat}}

## See also

- {{domxref("HTMLDataListElement")}}
- {{domxref("HTMLOptionElement")}}

> - {{htmlelement("datalist")}}

Reviewer: ```suggestion
- {{htmlelement("datalist")}}
- {{htmlelement("option")}}
```
